proto/motor/v1/response.proto

```diff
@@ -21,8 +21,8 @@ message InitializeResponse {
 
 message CreateAccountResponse {
   bytes aes_psk = 1;
-  string address = 2;
-  bytes did_document = 3;
+  bytes aes_dsc = 2;
```

Comment:
is it okay to plain text transport this key? I think this might be a security risk.

Reply:
+1

My refactor for keychain in motor will remove this from the request/response models anyway.